src/views/FileEntity/utils.tsx

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Temporarily ignoring warnings  because of the multiple TODOs for the missing data
![915FDF92-1A03-4812-81FF-9401C631EBAC_4_5005_c](https://user-images.githubusercontent.com/116835792/212245026-8a5f0b2b-9e71-4f60-bc9a-2feba74e60fa.jpeg)